Training/Training/file.c

```diff
@@ -1,13 +1,124 @@
 // ------------------------------------------------------------------------------------------------
 // Training ~ A training program for new joiners at Metamation, Batch - July 2024.
-// Copyright (c) Metamation India.
+// Gokul TS Copyright (c) Metamation India.
 // ------------------------------------------------------------------
-// Program.c
+// file.c
```

Comment:
This file is called harness.c. You should not make changes in a program given to you that are not necessary.